### ISSUE TITLE
[A11y] Desktop/Mobile Explore Collection Grid: Pagination Gray Previous Link

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1793]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

- [ ] Yes

### Issue Description

Explore Collections Grid: https://staging.oregondigital.org/explore_collections/all?locale=en&view=masonry

The pagination below shows previous using the following gray value #777777. It is not passing the color contrast for accessibility. Global issue across all pages that have pagination.

Path: 
a[rel="prev"]

Snippet:
`<a rel="prev" onclick="return false;" href="#"><i class="fa fa-chevron-circle-left"></i> Prev</a>`

### Suggested Fix

Fix the following:Element has insufficient color contrast of 4.47 (foreground color: #777777, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

I would suggest using #575757, this value is a darker gray that passes the color contrast with a contrast ratio of 7.2:1

**Comment by CGillen** [comment]
Fixed by #1987
